src/main/java/duke/commands/AddEventCommand.java

```diff
@@ -0,0 +1,36 @@
+package src.main.java.duke.commands;
+
+import src.main.java.duke.data.task.Event;
+
+
+/**
+ * Command that adds event
+ */
```

Comment:
Same comment about Java Documentation for AddDeadlineCommand.java. JDocs guidelines suggest header comments be in the form Returns (instead of Return or Returning). I noticed this issue in a few of your other classes as well, could it be due to a lapse in memory when refactoring your code?

Reply:
Did not have enough time. Will add more documentations. Thank you :)